packages/flutter_tools/lib/src/ios/ios_deploy.dart

```diff
@@ -336,10 +341,27 @@ class IOSDeployDebugger {
           .transform<String>(const LineSplitter())
           .listen((String line) {
         _monitorIOSDeployFailure(line, _logger);
+
+        // (lldb)    platform select remote-'ios' --sysroot
```

Comment:
Can you add a comment for why this check is needed, about the custom lldb prompt, etc?

Reply:
This LGTM once you add this comment

Reply:
Thx, i have added the comment, please have a look.